index.bs

```diff
@@ -5409,51 +5409,15 @@ ignored by the client browser or OS and not passed to the authenticator at all,
 Ignoring an extension is never considered a failure in WebAuthn API processing, so when [=[RPS]=] include extensions with any
 API calls, they MUST be prepared to handle cases where some or all of those extensions are ignored.
 
-Clients wishing to support the widest possible range of extensions MAY choose to pass through any extensions that they do not
-recognize to authenticators, generating the [=authenticator extension input=] by simply encoding the [=client extension input=]
-in CBOR. All [=WebAuthn Extensions=] MUST be defined in such a way that this implementation choice does not endanger the user's
+All [=WebAuthn Extensions=] MUST be defined in such a way that this implementation choice does not endanger the user's
 security or privacy. For instance, if an extension requires client processing, it could be defined in a manner that ensures such
```

Comment:
>Ignoring an extension is never considered a failure in WebAuthn API processing, so when [=[RPS]=] include extensions with any API calls, they MUST be prepared to handle cases where some or all of those extensions are ignored.
>
>All [=WebAuthn Extensions=] MUST be defined in such a way that this implementation choice does not endanger the user's security or privacy.

The start of the second paragraph ends up a non-sequitur without the text that was removed. I'll look over it and open a follow-up PR to fix it.

Reply:
Fixed in PR #1768.